src/main/resources/user.conf

```diff
@@ -44,3 +44,30 @@ peer.listen.port = 30303
 
 # Internal helper value, if database.dir is not set
 # database.name=database
+
+# Modules configuration
+modules {
```

Comment:
Maybe break it into:
```
modules {
  web = true
  rpc = true
  contracts = true
}

ports {
  web = 8080
  rpc = 8545
}
```

Reply:
I'd like to play it like modularity, but using just `port` instead of `listen.port` looks like a good idea